dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -607,10 +607,7 @@ private void initTabPane() {
 
         final Optional<Node> first = vbox.getChildren().stream().filter(i ->
                 i instanceof TreeTableView).findFirst();
-        if (first.isPresent()) {
-            return (TreeTableView<Ds3TreeTableValue>) first.get();
-        }
-        return null;
+        return (TreeTableView<Ds3TreeTableValue>) first.orElse(null);
```

Comment:
The orElse null kind of defeats the purpose of using an optional...  While this is better than the original version, why not return the Optional and let the caller deal with it?

Reply:
first can't be cast to Optional<TreeTableView<Ds3TreeTableValue>>